Presentation.Web/Models/API/V2/Response/SystemUsage/ItSystemUsageValidityResponseDTO.cs

```diff
@@ -18,6 +18,10 @@ public class ItSystemUsageValidityResponseDTO
         /// </summary>
         public bool ValidAccordingToLifeCycle{ get; set; }
         /// <summary>
+        /// Determines if the entity is considered valid based on the MainContract.IsActive
```

Comment:
why do you keep referring to domain objects in the comments... the api should not expose the underlying domain model because that may change and it only takes a second to make it out of data